editor/src/document/document_file.rs

```diff
@@ -493,7 +501,8 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 				)
 			}
 			SaveDocument => {
-				self.saved_document_identifier = self.current_identifier();
+				self.saved(true);
```

Comment:
`saved` also seems like a weird name for a void function. Could you use a few more words to describe what this function does?

Reply:
I changed it to `set_save_state` it is meant to be an abstraction over the way that we actually track whether a document is saved. @TrueDoctor has mentioned that it is generally bad in rust to use get/set. Is there a better name that you can think of?

Reply:
That's true for getters, not setters. https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
Ok, thanks for the clarification. Is the name change better?

Reply:
Yes, thanks